examples/spark/pipelines/spark_pipeline/spark_pipeline.py

```diff
@@ -0,0 +1,31 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+
+
+from zenml.pipelines import pipeline
+
+
+@pipeline
+def spark_pipeline(
+    importer,
+    analyzer,
+    splitter,
+    processer,
+    trainer,
+):
+    dataset = importer()
+    analyzer(dataset)
```

Comment:
this `analyzer` step feels a bit incomplete here, if there is a step like this in the pipeline I would naturally be curious to know how this can now be accessed/visualized. If we have some way to visualize the pandas `describe` as part of the post execution, then I would include that code in `run.py`, otherwise I would propose to remove this step.